checklist.html

```diff
@@ -27,8 +27,15 @@ <h2 id="toc_title" class="toc-title">Categories</h2>
 
   <fieldset class="article-section resources-section" id="aria-roles" aria-labelledby="label-landmarks" tabindex="-1">
 		<legend id="label-landmarks">Landmarks</legend>
-		<p>ARIA Landmark Roles are helpful landmarks that can be used by <abbr title="Assistive Technology">AT</abbr> to navigate a website.</p>
-		<p>Note: When you <a href="https://validator.w3.org/">validate html</a> using landmark roles, you'll receive a warning stating these roles are redundant.  In HTML5, several of the landmark roles are implicit via the native structural element which is supported by most modern <a href="http://stevefaulkner.github.io/html-mapping-tests/">desktop browsers</a> with the exception of IE and <a href="https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks-example.html">iOS Safari</a>. So, if you support IE and iOS browsers, you'll want to use the landmark roles. For more information, read <a href="/posts/aria-landmark-roles/">Quick Tip: Aria Landmark Roles and HTML5 Implicit Mapping</a>.</p>
+		<p>
+      ARIA Landmark roles can help assistive technology users to quickly navigate to and past blocks of content in a web interface.
+    </p>
+		<p>
+      <strong>Note:</strong> When you <a href="https://validator.w3.org/">validate HTML</a> that utilizes most landmark roles, you'll receive a warning stating these roles are redundant.
```

Comment:
Should we update this to use the [Nu checker](https://validator.w3.org/nu/)?

Reply:
yup.  was thinking about that last night, but clearly got distracted / didn't change